### PR TITLE
React to pipeline changes

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: src/Kestrel.Core/Internal/Infrastructure/IThreadPool.cs [hunk 1]
```diff
@@ -7,9 +7,9 @@
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure
 {
-    public interface IThreadPool : IScheduler
+    public abstract class IThreadPool: Scheduler
```
Comment (davidfowl) [inline src/Kestrel.Core/Internal/Infrastructure/IThreadPool.cs]: Rename to something else?

File: test/Kestrel.Core.Tests/HttpParserTests.cs [hunk 2]
```diff
@@ -2,8 +2,9 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Buffers;
 using System.IO.Pipelines;
-using System.IO.Pipelines.Testing;
+//using System.IO.Pipelines.Testing;
```
Comment (davidfowl) [inline test/Kestrel.Core.Tests/HttpParserTests.cs]: Remove dead code

File: src/Kestrel.Core/Internal/Infrastructure/IThreadPool.cs (hunk 1 above)
Comment (pakrym) [reply src/Kestrel.Core/Internal/Infrastructure/IThreadPool.cs]: Yep, I'll clean up the code, just wanted to see if I broke anything large during  refactorings.

File: NuGet.config [hunk 3]
```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="local" value="d:\localnuget" />
```
Comment (muratg) [inline NuGet.config]: temporary?

File: build/sources.props [hunk 4]
```diff
@@ -8,6 +8,8 @@
       https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json;
       https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json;
       https://dotnet.myget.org/F/roslyn/api/v3/index.json;
+      https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
+      https://dotnet.myget.org/f/dotnet-corefxlab/api/v3/index.json;
```
Comment (muratg) [inline build/sources.props]: temporary?

File: NuGet.config (hunk 3 above)
Comment (pakrym) [reply NuGet.config]: Yes, all feed changes in reaction PRs are temporary and would be undone before the commit.

**Pull request merged**